src/components/Person/Person.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person as PersonType } from '../../types';
+import { usePeoplePageContext }
+  from '../../pages/PeoplePage/PeoplePageContext/PeoplePageContext';
+
+type Props = {
+  person: PersonType
```

Comment:
You can rename this component to PersonTableRow or smth like that. No neen to rename types or create new

Reply:
> You can rename this component to PersonTableRow or smth like that. No need to rename types or create new I guess

It's ok